OPERATOR.rst

```diff
@@ -55,6 +55,27 @@ Getting started as operator
 Operator jobs
 -------------
 
+Check daily for OpenSearch Service Software Updates
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Amazon OpenSearch Service releases service software updates periodically, which
+are automatically applied after two weeks from the time a notification about the
+upgrade is received. The operator must prevent this upgrade from happening
+automatically and instead manually apply it through the AWS OpenSearch Service
+console when prod is idle.
+
+To check for upgrades:
+
+#. Go to https://us-east-1.console.aws.amazon.com/esv3/home?region=us-east-1#opensearch/notifications
```

Comment:
This is not how links are formatted in rst.

Anyways: don't paste deep links to the AWS console. They break in no time. Use English to describe where to go.